content/source/docs/plugin/framework/publishing.html.md

```diff
@@ -8,35 +8,26 @@ description: |-
 
 # Publishing to the Registry
 
-Publishing providers built on the provider development framework to the
-Terraform Registry works the same as [publishing providers built on
-SDKv2](/docs/registry/providers/publishing.html), with a minor difference.
-Providers built on the framework are built on version 6 of the Terraform
-protocol. The Registry, by default, assumes providers being published support
-version 5 of the Terraform protocol, which providers built on the framework do
-not. The Registry needs to be told about this difference in protocol version
-support. This is done by adding a manifest to the provider's release assets.
+You can publish providers to the Terraform Registry using the same [process as providers built on SDKv2](/docs/registry/providers/publishing.html), with one extra step.
+
+The Terraform Registry assumes that published providers support
+version 5 of the Terraform protocol, but providers built using the framework are built on version 6. You need to add a manifest to your provider's release assets that tells the Terraform Registry about this difference.
 
 ## Add a Version Manifest
 
-When uploading your new provider version to GitHub, next to the zip files
-containing your binaries for each platform, include a file named
-`terraform-provider-$NAME_$VERSION_manifest.json` (where `$NAME` is your
-provider's name, like `random` and `$VERSION` is your provider's version, like
-`1.2.3`) with the following contents:
+When you upload your new provider version to GitHub:
+
+1. Next to the zip files containing your binaries for each platform, include a file named `terraform-provider-$NAME_$VERSION_manifest.json` (where `$NAME` is your provider's name, like `random` and `$VERSION` is your provider's version, like `1.2.3`). Add the following contents:
 
-```
+    ```
 {
   "version": 1,
```

Comment:
I suggest using a realistic example for version, like 1.0.0

Reply:
I would also include the file name in the example block to show how the contents are reflected in the name

Reply:
@paddycarver is the `version` in this file different from the $VERSION used in the file name?

Reply:
I just pushed a change to make this "1.0.0" so that it's more realistic and matches the example for the name above. I also added text to remind folks that they shouldn't just paste it as is. @aareet What do you think?

Reply:
Version 1 is correct. This is a different version than the provider version, it's the version of the manifest format.

Reply:
Thanks @paddycarver! @aareet I pushed one final edit that puts the version number BACK to 1, per Paddy's explanation. But I did add a little explanation in the second step that this is the version of the manifest, not the version of the provider - hopefully this way folks won't get confused. Let me know if you feel comfortable approving after this change :-)